update.go

```diff
@@ -157,6 +159,13 @@ func (ub *UpdateBuilder) Limit(limit int) *UpdateBuilder {
 	return ub
 }
 
+// If sets IF expressions for UPDATE.
+func (ub *UpdateBuilder) If(andExpr ...string) *UpdateBuilder {
```

Comment:
It's not recommended to add this method. `IF` is a CQL-specific extension. AFAIK, there is no other DBMS supporting this. I recommend you to implement this in your CQL business code.

```go
func CQLUpdateIf(ub *sqlbuilder.UpdateBuilder, andExpr ...string) *sqlbuilder.UpdateBuilder {
    ub.SQL("IF")
    ub.SQL(strings.Join(andExprs, " AND "))
    return ub
}
```

As you can see, this package mainly defines methods for keywords available in ISO SQL-2016 spec, which is a subset of all DBMS implementations, with a few exceptions. I do think it's not convenient for users who build SQL with only one flavor. There is a long standing issue #60 (in Chinese) in which the support to `ON DUPLICATE KEY UPDATE` is requested. I cannot resolve this issue as SQL syntax for duplicate key differs a lot in different DBMS. It's hard to define a unified method for this feature.

A possible way to support flavor-specific keywords is to create sub-package for every flavor, e.g. `github.com/huandu/go-sqlbuilder/cql` for CQL, and then define methods based on the flavor's SQL spec. However, there will be a lot of work to do so.